tests/integration.test.ts

```diff
@@ -484,6 +492,28 @@ Deno.test("should handle json modules", async () => {
   });
 });
 
+Deno.test("should build project with another package manager", async () => {
+  await runTest("test_project", {
+    entryPoints: ["mod.ts"],
+    outDir: "./npm",
+    shims: {
+      ...getAllShimOptions(false),
+      deno: "dev",
+    },
+    package: {
+      name: "add",
+      version: "1.0.0",
+    },
+    packageManager: "yarn",
```

Comment:
So we don't need yarn installed for the tests, maybe we should find the absolute path of `"npm"` and then supply it here.

I've created this module to help us do that: https://github.com/dsherret/deno_which

Reply:
However, if we still use npm, how can we make sure it really works? Using absolute path to `"npm"` or just `"npm"` has no differences.

Reply:
True. Ok, let's leave this.